tools/package-builder/src/__test__/script/exec.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/env bash
+
+pnpm build
+pnpm i @platforma-open/milaboratories.runenv-python-3
```

Comment:
Why not to commit pnpm-lock file into the repo and keep runenv-python-3 as a dependency in package.json?
This would avoid situations, when test get broken because of incompatible changes in modern version of runenv-python-3 (if any appear).